src/main/kotlin/dev/restate/sdktesting/infra/RestateContainer.kt

```diff
@@ -46,24 +46,28 @@ class RestateContainer(
     private val WAIT_STARTUP_STRATEGY =
         WaitAllStrategy()
             .withStrategy(
-                Wait.forHttp("/restate/health")
-                    .forPort(RUNTIME_INGRESS_ENDPOINT_PORT)
+                Wait.forHttp("/metrics")
```

Comment:
`/metrics` will be reachable once the node grpc server starts up. Note at this point (if manual provisioning is enabled) the other services (ingress, admin, etc.) are not yet running.